activerecord/lib/active_record/persistence.rb

```diff
@@ -240,11 +240,15 @@ def upsert(attributes, returning: nil, unique_by: nil)
       #
       # ==== Examples
       #
-      #   # Insert multiple records, performing an upsert when records have duplicate ISBNs
+      #   # Given a Unique Index on books.isbn and the following record:
```

Comment:
"Unique Index" -> "unique index"

To be consistent with what is used elsewhere.